src/components/tabs/Tabs.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="tabs">
-    <ul class="tabs__controls" role="tablist">
+    <ul v-if="tabs.length > 1" class="tabs__controls" role="tablist">
```

Comment:
show tabs__controls only when there is more than one variant of the product.
Otherwise, it looks weird with just one tab 👇 
![image](https://user-images.githubusercontent.com/63985172/140860647-343de8f7-4e32-4963-aeea-d1382ef7966f.png)